lib/internal/util/inspect.js

```diff
@@ -177,7 +177,7 @@ const strEscapeSequencesReplacerSingle = /[\x00-\x1f\x5c\x7f-\x9f]/g;
 const keyStrRegExp = /^[a-zA-Z_][a-zA-Z_0-9]*$/;
 const numberRegExp = /^(0|[1-9][0-9]*)$/;
 
-const coreModuleRegExp = /^    at (?:[^/\\(]+ \(|)((?<![/\\]).+)\.js:\d+:\d+\)?$/;
+const coreModuleRegExp = /^    at (?:[^/\\(]+ \(|)node:((?<![/\\]).+):\d+:\d+\)?$/;
```

Comment:
This RegExp can probably now be simplified further by removing the lookbehind:
```suggestion
const coreModuleRegExp = /^    at (?:[^/\\(]+ \(|)node:(.+):\d+:\d+\)?$/;
```